app/src/main/java/com/portfolio/tasky/FieldsValidator.kt

```diff
@@ -0,0 +1,5 @@
+package com.portfolio.tasky
+
+interface FieldsValidator {
+    fun fieldsValidated(valid : Boolean)
```

Comment:
If you end up using UseCases (Explained on other comments)

This wouldn't be necessary. Since the usecases will return a Boolean themselves

Reply:
Okay.
Done.